docs/feature-gates.md

```diff
@@ -413,3 +414,8 @@ bandwidth for all egress traffic belonging to an Egress. Refer to this [document
 
 This feature leverages OVS meters to do the actual rate-limiting, therefore this feature requires OVS meters
 to be supported in the datapath.
+
+### EgressSeparateSubnet
+
+`EgressSeparateSubnet` allows users to allocate Egress IPs from a separate subnet different from the default Node
```

Comment:
"a different subnet from the default.."?

Reply:
done